packages/ragbits-core/src/ragbits/core/llms/base.py

```diff
@@ -66,8 +67,12 @@ async def generate_raw(
 
         Returns:
             Raw text response from LLM.
+        Raises:
+            ValueError: prompt containing images and model does not support vision
```

Comment:
As I mentioned during daily, I imagined this differently (handling the request sans the images, with a warning) but this is also okay. I have no problem with throwing an exception. But then we should create a more specific exception that is specifically for cases where the model doesn't have required capabilities, so that the code that calls the client can easily detect and handle such cases.

Reply:
warnings are raised on level of llm client now